silo-vaults/contracts/MetaMorphoFactory.sol

```diff
@@ -31,8 +39,13 @@ contract MetaMorphoFactory is IMetaMorphoFactory {
         string memory symbol,
         bytes32 salt
     ) external returns (IMetaMorpho metaMorpho) {
-        metaMorpho =
-            IMetaMorpho(address(new MetaMorpho{salt: salt}(initialOwner, initialTimelock, asset, name, symbol)));
+        VaultIncentivesModule vaultIncentivesModule = VaultIncentivesModule(
+            Clones.clone(VAULT_INCENTIVES_MODULE_IMPLEMENTATION)
```

Comment:
[cloneDeterministic](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/Clones.sol#L74) will allow to predict incentives module address.
```suggestion
            Clones.cloneDeterministic(salt, VAULT_INCENTIVES_MODULE_IMPLEMENTATION, value)
```

Reply:
why we need to predict that?

Reply:
imo prediction here is useless, I see no reason why we want or need that for creating vaults.

Reply:
https://github.com/silo-finance/silo-contracts-v2/pull/882#discussion_r1888159024